confgenerator/testdata/valid/linux/logging-receiver_forward/input.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+logging:
+  receivers:
+    fluent_logs:
```

Comment:
Should we unify the receiver / pipeline ID as well for the forward receiver? Similar to https://github.com/GoogleCloudPlatform/ops-agent/pull/384#issue-1123802764

aka `s/fluent_logs/fluent_forward` and `s/fluent_pipeline/fluent_forward`